ideas.html

```diff
@@ -1 +1,5 @@
-
+<h1>Lily idea #1</h1>
+<p>
+  You create your sandwich by draging items into the sandwich anyway you want. Then you place your order and pay.
+  then it asks you if you wnat to play sandwich games. If you get the high score you get 10% of the next sandwich you buy!
```

Comment:
How do we store the high score?

Reply:
Ya how? I have no clue.

Reply:
IDK.

Reply:
Maybe we could look at some other games codes and see how they did that. But like Jasper I have no clue how to do that!

Reply:
We'll have to use Javascript and then have our own server to store the information

Reply:
I didn't understand anything you just said

Reply:
I mean I know what Javascript is